dev/TabView/TabViewItem.cpp

```diff
@@ -236,13 +334,25 @@ void TabViewItem::OnPointerEntered(winrt::PointerRoutedEventArgs const& args)
     {
         m_isMiddlePointerButtonPressed = true;
     }
+
+    // If we show close button onhover, we now must show the close button
+    if (m_closeButtonOverlayMode == winrt::TabViewCloseButtonOverlayMode::OnHover)
+    {
+        UpdateCloseButtonVisibility(winrt::Visibility::Visible);
+    }
```

Comment:
This also needs to go through UpdateCloseButton() -- that should be the only place where the close button visibility is changed.